indexing-service/src/main/java/org/apache/druid/indexing/overlord/http/OverlordResource.java

```diff
@@ -267,6 +267,7 @@ public Response getTaskStatus(@PathParam("taskid") String taskid)
                   // TaskStorage API doesn't yet allow it.
                   DateTimes.EPOCH,
                   taskInfo.getStatus().getStatusCode(),
+                  taskInfo.getStatus().getStatusCode(),
```

Comment:
Let's add a new constructor for TaskStatusPlus without the `@JsonCreator` annotation that only has "statusCode" which automatically fills in the deprecated "status" parameter with the same value

Reply:
yes, that's a good suggestion, should we make the existing constructor private as well

Reply:
Nah that's fine, I don't think it can be private anyway since it's the `@JsonCreator`